todo/ios/Podfile

```diff
@@ -3,11 +3,12 @@ platform :ios, '9.0'
 target 'MongoDBSample' do
   use_frameworks!
 
-  pod 'Google/SignIn'
+  pod 'Firebase/Core'
```

Comment:
Is this necessary? See what happens if you remove this, run `pod install` and try to build again.